app/services/validation_beta_service.rb

```diff
@@ -172,6 +172,67 @@ def sit_with_unvalidated_participants_reminders
     end
   end
 
+  def sit_fip_with_unvalidated_participants_deadline_reminders
+    campaign = :unvalidated_participants_deadline_reminder
+
+    sign_in_url = Rails.application.routes.url_helpers.new_user_session_url(
+      host: Rails.application.config.domain,
+      **UTMService.email(campaign, campaign),
+    )
+
+    participant_start_url = Rails.application.routes.url_helpers.participants_start_registrations_url(
+      host: Rails.application.config.domain,
+      **UTMService.email(campaign, campaign),
+    )
+
+    InductionCoordinatorProfile
+      .joins(schools: %i[active_ecf_participant_profiles partnerships])
+      .includes(schools: { active_ecf_participant_profiles: %i[ecf_participant_eligibility ecf_participant_validation_data] })
+      .where(
+        school_cohorts: {
+          cohort_id: Cohort.current.id,
+          induction_programme_choice: "full_induction_programme",
+        },
+        ecf_participant_eligibility: {
+          participant_profile_id: nil,
+        },
+        ecf_participant_validation_data: {
+          participant_profile_id: nil,
+        },
+      )
+      .merge(Partnership.active)
+      .distinct
+      .find_each do |sit|
+        sit.schools
+          .joins(:school_cohorts, :partnerships)
+          .where(school_cohorts: { cohort_id: Cohort.current.id })
+          .merge(Partnership.active)
+          .find_each do |school|
+          participants = school
```

Comment:
I think if a SIT has two partnered FIP schools, and only one has unvalidated participants this will sent an email with an empty list for one of the schools

Reply:
It's a good point, not sure what the chances are but I'll add something in as you suggest